src/js/node.rs

```diff
@@ -0,0 +1,220 @@
+use std::{collections::HashMap, path::PathBuf};
+
+use crate::{
+    ConnType, ConnectionTypeMixed, CounterStats, DirectAddrInfo, IrohNode, NodeStatusResponse,
+    PublicKey,
+};
+
+use futures::TryStreamExt;
+use napi_derive::napi;
+
+#[napi]
+impl DirectAddrInfo {
+    /// Get the reported latency, if it exists, in milliseconds
+    #[napi(js_name = "latency")]
+    pub fn latency_js(&self) -> Option<u32> {
```

Comment:
(in general) should we use the napi bigint type instead of u32 if we already have a javascript specific version, that way we don't screw up precision

Reply:
well well well, turns out you can just return u64 & napi will convert that to a `BigInt` for you...

Also, refactored for that we take a `BigInt` anywhere that the normal rust api allows a u64